### DISCUSSION TITLE
volume knob

### DISCUSSION THREAD

**Discussion by treefrogman** [maximecb/noisecraft#97] [Ideas]
When I [browse](https://noisecraft.app/browse) through the (often magnificent) creations, I find I have to carry a volume knob setup in my clipboard and insert it in each creation before listening, as my audio setup goes through my TV and the volume adjuster is crap.

I propose that, like the Clock node, the AudioOut node get a built-in knob: for attenuating the output. While it would render some conscientious creators' final Mul Knobs superfluous, I think the benefit would outweigh the redundancy.

This is not without precedent either, as every online media player I can think of has a built-in volume control.

**Comment by maximecb** [comment]
I'm not completely opposed to it, but it does go a bit against the idea of keeping NoiseCraft minimalistic.

Q: why are you not able to adjust the volume easily on your setup? Are you feeding an HDMI cable into your TV and using the TV remote for volume control?

The slight annoyance here is that if the AudioOut node had a volume knob, you might still have to go and locate the AudioOut node, which could take you a few seconds on some larger projects 😅 

Again, not saying no, but give me your most convincing arguments.

**Reply by treefrogman** [reply]
Regarding your question, the answer actually changes a lot. Many of the audio systems I find myself using don't have good volume controls. I think a recognition of this common problem is why most media player apps have their own built-in volume controls.

You make a good point about hunting it down. What if the play button had a volume slider to the left of it?

It could default to full volume and then save your settings to localStorage and use that when opening a new NoiseCraft window.
